app/scripts/components/shared/new-button.components.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+
+class Button extends React.Component {
+	render() {
+		const {
+			size,
+			children,
+			className,
+			...rest
+		} = this.props;
+
+		const classes = classnames(
+			'new-button',
+			{
+				'new-button--tiny': size === 'tiny',
```

Comment:
I'm building a new button as the previous ones were not greatly independent. The goal is to have a generic component that's usable everywhere without having to create one every time.
In addition, as it's possible to pass down a `className` prop, we can override it whenever we need it. (and it's also in the storybook, so we can't mess up with it without knowing)